src/main/show-notification/show-notification.injectable.ts

```diff
@@ -3,11 +3,19 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 import { getInjectable } from "@ogre-tools/injectable";
+import applicationWindowInjectable from "../start-main-application/lens-window/application-window/application-window.injectable";
+import { notificationChannel } from "../../common/notification/notification-channel";
 
 const showNotificationInjectable = getInjectable({
   id: "show-notification",
 
-  instantiate: (di) => (message: string) => {},
+  instantiate: (di) => {
+    const applicationWindow = di.inject(applicationWindowInjectable);
+
+    return (message: string) => {
+      applicationWindow.send({ channel: notificationChannel.name, data: [message] });
+    };
+  },
 });
```

Comment:
I don't think this should be an injectable on main. It hard codes too much into `main`.

Reply:
This is removed in latter commit. 

However, I do see this as important thing that might be reintroduced in the future. There is no point of not letting the `main` process code be written with actual behaviours since that 's just cleaner code.